dojo/tools/spotbugs/parser.py

```diff
@@ -86,41 +82,52 @@ def get_findings(self, filename, test):
             for message in bug.itertext():
                 desc += message + '\n'
 
-            dupe_key = bug.get('instanceHash')
-
-            title = bug.find('ShortMessage').text
+            shortmessage_extract = bug.find('ShortMessage')
+            if shortmessage_extract:
+                title = shortmessage_extract.getText()
```

Comment:
Something isn't working with the title in my test. Will investigate tomorrow, no time today anymore. For now just want to make sure the PR is not merged.

Reply:
The title in my testing appears to be fine. The new scan (4.4.0) in this PR does not use `ShortMessage`, but the[ other sample scan ](https://github.com/DefectDojo/django-DefectDojo/blob/master/dojo/unittests/scans/spotbugs/many_findings.xml)does. Here's a screenshot of both versions titles.

version 4.4.0
<img width="717" alt="image" src="https://user-images.githubusercontent.com/46459665/132784050-4eb5b3af-a89a-4f16-9c97-2aae3b42ef35.png">

Version 4.2.x
<img width="717" alt="image" src="https://user-images.githubusercontent.com/46459665/132784105-94c1678b-ab47-4284-942d-9c04dde6eb1f.png">

Reply:
In the version 4.2 example you see the problem. The title of the first one should be **Potential Command Injection** which is the content of `ShortMessage`.  

It works for me when I change the detection of the title to:

```
shortmessage_extract = bug.find('ShortMessage')
if shortmessage_extract is not None:
    title = bug.find('ShortMessage').text
else:
    title = bug.get('type')
```

Reply:
The unit tests for the old version of the reports doesn't have a test for the title. With such a test you would have noticed the title has changed with this fix to a poorer version.

Reply:
I added unit tests for title and saw the pb. You were right @StefanFl , I fixed my PR